08-projects.Rmd

```diff
@@ -6168,34 +6195,34 @@ If you paste a `title_id` to the end of the following url, it will pull up the p
 - Output of the query.
 ```
 
-##### 4. The `episode_title_id` column in the `episodes` table references titles of individual episodes of a tv series. The `show_title_id` references the titles of the show itself. With that in mind, write a query that gets a list of all of the episodes and titles of Grey's Anatomy.
+##### 4. The `episode_title_id` column in the `episodes` table references titles of individual episodes of a TV series. The `show_title_id` references the titles of the show itself. With that in mind, write a query that gets a list of all of the episodes and their titles of _Grey's Anatomy_.
 
-**Relevant topics:** [inner join](#sql-inner-join)
+**Relevant topics:** [INNER JOIN](#sql-inner-join)
 
 ```{block, type="bbox"}
 **Item(s) to submit:**
 
 - SQL query used to solve the problem in a code chunk.
 ```
 
-##### 5. Like we explained in (3), you can find the `title_id` of a tv show, a tv show episodes, or a movie by browsing imdb.com and getting the `title_id` directly from the url. Browse imdb.com and find your favorite tv show. Get the `title_id` from the url and run the following query to confirm that the tv show is in our database:
+##### 5. Like we explained in (3), you can find the `title_id` of a TV show, a TV show episodes, or a movie by browsing imdb.com and getting the `title_id` directly from the url. Browse imdb.com and find your favorite TV show. Get the `title_id` from the url and run the following query to confirm that the TV show is in our database:
 
 ```{sql, eval=F}
 SELECT * FROM titles WHERE title_id='<title id here>';
 ```
 
-Make sure to replace "\<title id here\>" with the `title_id` of your favorite show. If your show does not appear, or has only a single season, pick another show until you find one we have in our database with multiple seasons.
+Make sure to replace "\<title id here\>" with the `title_id` of your favorite show. If your show does not appear, pick another show until you find one we have in our database.
 
 ```{block, type="bbox"}
 **Item(s) to submit:**
 
-- The `title_id` of your favorite tv show.
+- The `title_id` of your favorite TV show.
 - The output from running the provided (modified) query.
 ```
 
-##### 6. We want to write a query that returns the title and rating of the highest rated episode of the tv show you chose in (5). In order to do so, first write a query that returns a list of `episode_title_id`s (found in the `episodes` table), with the `primary_title` (found in the `titles` table) of the episode. 
+##### 6. We want to write a query that returns the title and rating of the highest rated episode of the TV show you chose in (5). In order to do so, we will break the task into two parts in (6) and (7). First, write a query that returns a list of `episode_title_id`s (found in the `episodes` table), with the `primary_title` (found in the `titles` table) of the episode. 
```

Comment:
The addition was intended to give a heads-up about the relationship between (6) and (7).

Reply:
Thanks!